data.sql

```diff
@@ -6,3 +6,71 @@ VALUES('Agumon', '2020-02-03', 0, true,10.23),
      ('Gabumon', '2018-11-15', 2, true,8),
      ('Pikachu', '2021-01-07', 1, false,15.04),
      ('Devimon', '2017-05-12', 5, true,11);
+
+
+-- Query and Update animals table
+
+
```

Comment:
- [ ] Kindly update the data as it's requested on the requirements 

![image](https://github.com/Bleu-Yves-Sopoude/vet-clinic-db/assets/93630700/75e1323a-d7db-4dc8-964e-77d54c725f3e)